nixos/tests/containers-tmpfs.nix

```diff
@@ -0,0 +1,45 @@
+# Test for NixOS' container support.
+
+import ./make-test.nix ({ pkgs, ...} : {
+  name = "containers-bridge";
+  meta = with pkgs.stdenv.lib.maintainers; {
+    maintainers = [ ckampka ];
+  };
+
+  machine =
+    { config, pkgs, ... }:
+    { imports = [ ../modules/installer/cd-dvd/channel.nix ];
+      virtualisation.writableStore = true;
+      virtualisation.memorySize = 768;
+
+      containers.webserver =
+        {
+          autoStart = true;
+          privateNetwork = true;
+          tmpfs = [ "/var" ];
```

Comment:
Somehow I think the test should do more then just mounting `/var` on tmpfs. What about mounting `/var/log` and `/var/www` on tmpfs, then place a file in `/var/www/` inside the container and check that its not there when looking at `/var/lib/containers/webserver/var/www` from the outside?

Also why do you run an httpd when your test does nothing with it. Maybe remove that.

Also what happens when you mount a directory on tmpfs where the path is not present? Something like `/var/lib/ridiculous`?